src/v/cluster/controller.cc

```diff
@@ -380,4 +381,39 @@ ss::future<> controller::stop() {
     });
 }
 
+/**
+ * This function provides for writing the controller log immediately
+ * after it has been created, before anything else has been written
+ * to it, and before we have started communicating with peers.
+ */
+ss::future<> controller::cluster_creation_hook() {
+    if (!config::node().seed_servers().empty()) {
+        // We are not on the root node
+        co_return;
+    } else if (
+      _raft0->last_visible_index() > model::offset{0}
```

Comment:
this has to be `_raft0->last_visible_index() > model::offset{}`

Reply:
You're right, fixed.